src/predict/gcp_helpers/cloud_function.py

```diff
@@ -122,15 +123,22 @@ def predict_model(m):
     locations_flat = np.c_[locations[0].flatten(),locations[1].flatten()]
     pred_set = np.c_[locations_flat,np.full(locations_flat.shape[0],time)]
     mean, var = m.predict_f(pred_set)
-
+    
     means = mean.numpy().flatten()
     variances = var.numpy().flatten()
+    std_dev = np.sqrt(variances)
+    # calculate prediction interval
+    interval = 1.96 * std_dev
```

Comment:
Hi @Priscilla-AL is `1.96` always a constant value or expected to change sometime in the future?

Reply:
okay...no worries, I see its still work in progress